.github/workflows/unit-tests.yml

```diff
@@ -7,6 +7,10 @@ on:
         description: 'If the theme is a vip theme'
         required: true
         type: boolean
+      WP_CORE_DIR:
```

Comment:
This isn't needed here. The `WP_CORE_DIR` envar is set in the composite action: https://github.com/penske-media-corp/github-action-wordpress-test-setup/blob/7da08df90b6c840971336eed47d53ed8046d8503/scripts/set-wordpress-envars.sh#L6. It carries over because we don't remove it from the `GITHUB_ENV` variable: https://github.com/penske-media-corp/pmc-artnews-2019/actions/runs/3415340677/jobs/5684360022#step:4:202

Reply:
updated